lib/roby/interface/client.rb

```diff
@@ -297,6 +317,31 @@ def call(path, m, *args)
 
             # @api private
             #
+            # Asynchronously call a method on the interface or on one of the
+            # interface's subcommands
+            #
+            # @param [Array<String>] path path to the subcommand. Empty means on
+            #   the interface object itself.
+            # @param [Symbol] m command or action name. Actions are always
+            #   formatted as action_name!
+            # @param [Object] args the command or action arguments
+            def async_call(path, m, *args, &block)
+                if m.to_s =~ /(.*)!$/
+                    action_name = $1
+                    if find_action_by_name(action_name)
+                        path = []
+                        m = :start_job
+                        args = [action_name, *args]
+                    else raise NoSuchAction, "there is no action called #{action_name} on #{self}"
+                    end
+                end
+                io.write_packet([path, m, *args])
+                pending_async_calls << { block: block, path: path,
+                                         m: m, args: [*args] }
```

Comment:
`[*args]` in this case is strictly the same as `args.dup` and since `args` is locally created in the method, it's functionally equivalent to plain `args`

Reply:
Done... 👍